From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Sat, 2 Mar 2024 22:59:31 +0100
Subject: [PATCH] HACK: allow feature build_hasher_simple_hash_one

In rustc 1.70.0 from OSELAS.Toolchain-2023.07.*, this feature is still
unstable, so explicitly allow it to make it possible to use this
toolchain.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 ptxdist-cargo-home/source/ahash-0.8.9/src/lib.rs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ptxdist-cargo-home/source/ahash-0.8.9/src/lib.rs b/ptxdist-cargo-home/source/ahash-0.8.9/src/lib.rs
index 653c3bc1e88a..edb2eebc9d26 100644
--- a/ptxdist-cargo-home/source/ahash-0.8.9/src/lib.rs
+++ b/ptxdist-cargo-home/source/ahash-0.8.9/src/lib.rs
@@ -12,6 +12,7 @@
 //! Because aHash does not have a fixed standard for its output, it is able to improve over time.
 //! But this also means that different computers or computers using different versions of ahash may observe different
 //! hash values for the same input.
+#![feature(build_hasher_simple_hash_one)]
 #![cfg_attr(
     all(
         feature = "std",
